src/components/IGVBrowser.test.js

```diff
@@ -76,23 +86,43 @@ jest.mock('src/workspaces/common/requester-pays/RequesterPaysModal', () => {
   return { __esModule: true, RequesterPaysModal };
 });
 
+const localStorageMock = (() => {
+  let store = {};
+  return {
+    getItem: jest.fn((key) => store[key] || null),
+    setItem: jest.fn((key, value) => {
+      store[key] = value.toString();
+    }),
+    removeItem: jest.fn((key) => {
+      delete store[key];
+    }),
+    clear: jest.fn(() => {
+      store = {};
+    }),
+  };
+})();
+
+Object.defineProperty(window, 'localStorage', {
+  value: localStorageMock,
+});
+
 afterEach(() => {
   jest.clearAllMocks();
 });
 
-describe('IGVBrowser', () => {
-  const mockWorkspace = {
-    workspace: { googleProject: 'test-project' },
-  };
+const mockWorkspace = {
+  workspace: { googleProject: 'test-project' },
+};
 
-  const mockSelectedFiles = [
-    {
-      filePath: 'gs://bucket/test.bam',
-      indexFilePath: 'gs://bucket/test.bai',
-      isSignedUrl: false,
-    },
-  ];
+const mockSelectedFiles = [
```

Comment:
I just moved these outside of the `IGVBrowser` group to make them available for the test group I added.